src/applications/disability-benefits/781/helpers.js

```diff
@@ -108,3 +170,57 @@ export const documentDescription = () => (
     </p>
   </div>
 );
+
+const UploadExplanation = ({ formType }) => (
+  <div>
+    <p>
+      If you have already completed a Claim for Service Connection for
+      Post-Traumatic Stress Disorder (VA Form 21-0
+      {`${formType}`}
+      ), you can upload it here instead of answering the questions about your
+      PTSD.
+    </p>
+    <p>How would you like to provide information about your PTSD?</p>
+  </div>
+);
+
+export const UploadPtsdDescription = ({ formData, formType }) => {
+  const { incidentText } = getPtsdClassification(formData, formType);
```

Comment:
Do we gain anything by doing it this way as opposed to 

```
<p>
         The following questions will help us understand more about your
         {` ${getPtsdClassification(formData, formType)}`}
         -related PTSD. None of the questions we‘ll ask you are required, but any
         information you provide here will help us research your claim.
       </p>
```

?

Reply:
@igreulich The way `getPtsdClassification` function is written as of now, it returns an object with the title and the text, there are some minor differences between them. I could inline it but then it would be .notation after a function. (example: `{getPtsdClassification(formData, formType).incidentText}`).

We could refactor in a way to return the title, vs the text.

Reply:
No worries, I was just curious.